### PR TITLE
FeelUOwn Survey 20230806

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

This topic surveys for FeelUOwn updates to fix compatibility issue with current MPV versions. It also introduces `feeluown-bilibili` and drops the unmaintained/merged `feeluown-local` and `feeluown-xiami`.

Ref: https://github.com/linuxmint/hypnotix/issues/254

Package(s) Affected
-------------------

Updated:

- `feeluown` v3.8.13
- `feeluown-bilibili` v0.2.0
- `feeluown-kuwo` v0.2.1
- `feeluown-netease` v0.9.7
- `feeluown-qqmusic` v0.5.0
- `pydantic` v1.10.2
- `qasync` v0.24.0
- `typing-extensions` v4.7.1

Dropped:

- `feeluown-local`, now part of `feeluown`
- `feeluown-xiami`, no longer maintained by upstream and is now incompatible

Security Update?
----------------

No

Build Order
-----------

```
pydantic qasync typing-extensions feeluown{,-{bilibili,kuwo,netease,qqmusic}}
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by Icenowy** [comment]
Tested playback with Netease.

File: lang-python/qasync/autobuild/defines [hunk 1]
```diff
@@ -1,8 +1,8 @@
 PKGNAME=qasync
 PKGSEC=python
-PKGDES="Python library for PyQt/PySide applications' async support"
-PKGDEP="python-3 pyside2 pyqt5"
+BUILDDEP="pyside2 pyqt5"
```
Comment (Icenowy) [inline lang-python/qasync/autobuild/defines]: Why dual BUILDDEP?

Comment (MingcongBai) [reply lang-python/qasync/autobuild/defines]: Oops

**Review by Icenowy — APPROVED** [review]
LGTM.

**Pull request merged**